task-sdk/src/airflow/sdk/definitions/_internal/abstractoperator.py

```diff
@@ -31,13 +31,13 @@
 import methodtools
 
 from airflow.configuration import conf
+from airflow.sdk.api.datamodels._generated import WeightRule
```

Comment:
I do not love this import pattern, IDK if its feasible to add it to lazy imports and have the usage be: `from airflow.sdk import WeightRule`?